cli/remotebazel/remotebazel.go

```diff
@@ -67,6 +67,7 @@ var (
 	remoteRunner   = remoteFlagset.String("remote_runner", defaultRemoteExecutionURL, "The Buildbuddy grpc target the remote runner should run on.")
 	timeout        = remoteFlagset.Duration("timeout", 0, "If set, requests that have exceeded this timeout will be canceled automatically. (Ex. --timeout=15m; --timeout=2h)")
 	execPropsFlag  = bbflag.New(remoteFlagset, "runner_exec_properties", []string{}, "Exec properties that will apply to the *ci runner execution*. Key-value pairs should be separated by '=' (Ex. --runner_exec_properties=NAME=VALUE). Can be specified more than once. NOTE: If you want to apply an exec property to the bazel command that's run on the runner, just pass at the end of the command (Ex. bb remote build //... --remote_default_exec_properties=OSFamily=linux).")
+	interactive    = remoteFlagset.Bool("interactive", true, "Whether to run the script in interactive mode (will print real-time progress updates using ANSI escape sequences). This may cause duplicate logs to be printed if the console does not support ANSI escape sequences.")
```

Comment:
Instead of making this an explicit flag, I think it would be better to check whether the command's output is a terminal or not. You can use `terminal.IsTTY(os.Stderr) && terminal.IsTTY(os.Stdout)` to check (from the `cli/terminal` package). When running locally in a terminal, this should return true. If you send the output to a pipe, like `bb remote ... | cat`, it'll return false. On GH actions, I think it will return false, since the actions runner is not running actions inside an interactive terminal.

Reply:
Done